package.json

```diff
@@ -3,30 +3,30 @@
   "scripts": {
     "postinstall": "npm run typings:install",
     "typings:install": "typings install",
-    "posttypings:install": "pbiviz update 1.13.0",
+    "posttypings:install": "pbiviz update 2.22.0",
```

Comment:
There are no version of API 2.22
Please use 2.2.0

Reply:
done